.coderabbit.yaml

```
@@ -0,0 +1,70 @@
+    scope: auto
+  pull_requests:
+    scope: auto
+
+# Code generation settings
+code_generation:
+  docstrings:
+    path_instructions:
+      - path: "**/*.js"
+        instructions: |
+          End all docstrings with a notice that says "Auto-generated by CodeRabbit.".
+          Do not omit the closing tags; the docstring must be valid.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)